charts/hush-sensor/templates/daemonset.yaml

```diff
@@ -85,3 +85,13 @@ spec:
           volumeMounts:
             - name: vector-socket
               mountPath: /tmp/vector
+          env:
+            - name: EVENT_REPORTING_ENDPOINT
+              {{- with (include "hush-sensor.deploymentInfo" . | fromYaml) }}
```

Comment:
`with` also checks if the value is "truthy". If it is not, the whole block isn't executed. So, it makes sense to put `-name: EVENT_...` inside the block too.